Blish HUD/GameServices/ContentService.cs

```diff
@@ -101,21 +107,39 @@ public enum FontStyle {
         }
 
         public Microsoft.Xna.Framework.Content.ContentManager ContentManager => Blish_HUD.BlishHud.ActiveContentManager;
-        
+
         protected override void Initialize() { /* NOOP */ }
 
         protected override void Load() {
             Textures.Load();
+            var zipArchiveReader = new ZipArchiveReader(RefPath);
+            _audioDataReader = zipArchiveReader.GetSubPath("audio");
```

Comment:
What was the reason for limiting it all down to the audio subpath and replacing the audio paths in all of the other places instead of it all still being relative to root?  Just figured it best to lock it down to all be under /audio?

I'm not against it, just wanted to know if there was anything else behind the decision.  👍

Reply:
Otherwise you always have a redundancy for the audio path and something that wants to access audio shouldn't care in which folder it resides in and let it handle from the called function. It makes more sense when the function is called "PlayAudioFromName" that you actually only need to put in a name and not a path. tldr: the caller of that function shouldn't care about a "filesystem" behind playing the audio.